src/m365/teams/commands/channel/channel-add.ts

```diff
@@ -64,8 +99,12 @@ class TeamsChannelAddCommand extends GraphCommand {
   public options(): CommandOption[] {
     const options: CommandOption[] = [
       {
-        option: '-i, --teamId <teamId>',
-        description: 'The ID of the team to add the channel to'
+        option: '-i, --teamId [teamId]',
+        description: 'The ID of the team to add the channel to. Specify either teamId or teamName but not both'
+      },
+      {
+        option: '-t --teamName [teamName]',
```

Comment:
I removed the short `-t` since it's ambiguous between `teamId` and `teamName`. I saw that we're using it in another command, but for now let's skip it. If need be, we can always add it later.